includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1045,7 +1045,7 @@ public function set_payment_method_title_for_order( $order, $payment_method_type
 	 * @return string
 	 */
 	public function generate_upe_checkout_experience_accepted_payments_html( $key, $data ) {
-		$stripe_account      = WC_Stripe_API::retrieve( 'account' );
+		$stripe_account      = $this->stripe_request( 'account' );
 		$stripe_capabilities = isset( $stripe_account->capabilities ) ? (array) $stripe_account->capabilities : [];
```

Comment:
I really think this should be handled using the `WC_Stripe_Account` class, which is initialised and stored as a property of `WC_Stripe`, since this class has better error handling and stores the response in a transient...but that's going to have to be a story for another day.